content/integrations/lambdatest-integration.md

```diff
@@ -0,0 +1,81 @@
+
+## Real Time and App Automation
+
+You can test your **.ipa** and **.apk** directly on real devices by submitting them to the **LambdaTest** environment via a cURL request:
+
+```
+-Scripts:
+    - name: Submitting app to LambdaTest:
+    script: |
+        curl --location --request POST 'https://manual-api.lambdatest.com/app/upload/realDevice' --header 'Authorization: Basic $LAMBDATEST' --form 'name="lambda1"' --form 'appFile=@"app/build/outputs/apk/release/app-release.apk"'
+```
+
+
+Base64 encoding the username and token [here](https://mixedanalytics.com/knowledge-base/api-connector-encode-credentials-to-base-64/) with the **username:accessToken** format is needed, then save the encoded string to Codemagic as an environment variable. So the steps would be as follow:
+
+1. Encode and save the username and access token into an environment variable.
+2. Run the scripts in YAML referencing that environment variable.
+
+**$LAMBDATEST** environment variable is a custom name and can be named to anything preferred. More info can be found [here](https://docs.codemagic.io/variables/environment-variable-groups/) about how to set up environment variable groups with Codemagic.
+
+As soon as your **.ipa** and **.apk** are successfully built, they will appear in the **LambdatTest UI** under **Real Device => Real Time**. Any preferred devices can be selected for testing with **Real Time**. 
+
+In order to see your tests being uploaded to the **App Automation**, tests need to be included in your project. As soon as tests are detected, they will be automatically uploaded to the **App Automation** section and all the results can be viewed there. However, in order to enable it, some capabilities must be injected into your project's test scripts:
+
+**For Android**:
+
+```    
+    DesiredCapabilities capabilities = new DesiredCapabilities();
+    capabilities.setCapability("platformName", "Android");
+    capabilities.setCapability("deviceName", "Google Pixel 3");
+    capabilities.setCapability("isRealMobile", true);
+    capabilities.setCapability("platformVersion","10");
+    capabilities.setCapability("app","lt://APP100202151634649275590734");
+    capabilities.setCapability("deviceOrientation", "PORTRAIT");
+    capabilities.setCapability("console",true);
+    capabilities.setCapability("network",true);
+    capabilities.setCapability("visual",true);
+```
+
+**For iOS**:
+
+```
+    DesiredCapabilities capabilities = new DesiredCapabilities();
+    capabilities.setCapability("platformName", "iOS");
+    capabilities.setCapability("deviceName", "iPhone 10");
+    capabilities.setCapability("isRealMobile", true);
+    capabilities.setCapability("platformVersion","10");
+    capabilities.setCapability("app","lt://APP100202151634649275590734");
+    capabilities.setCapability("deviceOrientation", "PORTRAIT");
+    capabilities.setCapability("console",true);
+    capabilities.setCapability("network",true);
+    capabilities.setCapability("visual",true);
+
+```
```

Comment:
Mention the file where this is set.

Reply:
Hmm, this should be set in a test script and that test script file could be located differently, I guess. Should we just leave it as a test script rather than showing an exact file path ?